spec/models/support/requests/brexit_feedback_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require "gds_api/support_api"
+require 'gds_api/test_helpers/support_api'
+
+module Support
+  module Requests
+    include GdsApi::TestHelpers::SupportApi
+
+    describe BrexitFeedbackRequest do
```

Comment:
This tests the happy path, given we use a lot of external services in this PR we should also test what happens with common endpoint failure cases.